### DISCUSSION TITLE
Custom binding address and port

### DISCUSSION THREAD

**Discussion by p7e4rs** [Athou/commafeed#1612] [Q&A] [answered]
Hello, I currently use ufw to disable direct port access (because it binds to 0.0.0.0:8082 by default), and use nginx to proxy localhost:8082. It currently works but is cumbersome. Is there a more convenient way, such as a setting option to specify commafeed's binding address and listening port?

**Comment by Athou** [comment, marked as answer]
Those two settings should let you change what you want:
https://quarkus.io/guides/all-config#quarkus-vertx-http_quarkus-http-host
https://quarkus.io/guides/all-config#quarkus-vertx-http_quarkus-http-port

**Reply by p7e4rs** [reply]
It worked for me, thank you!!
